calcom/cal.com
# feat: add instant meeting parameter

## What does this PR do?



- Fixes https://github.com/calcom/cal.com/issues/17660 (GitHub issue number)
- Fixes CAL-4734 (Linear issue number - should be visible at the bottom of the GitHub issue description)



![Screenshot 2024-11-15 at 6 08 16 PM](https://github.com/user-attachments/assets/2d79bcae-8da8-418b-8ee1-4930f1608832)


## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

1) Create a team org event types
2) Enabled instant bookings and add a parameter value and click save
3) Go to booking page and you should only see the modal when all parameter values are present
